perfmetrics/scripts/testing_on_gke/examples/utils/utils.py

```diff
@@ -155,6 +155,35 @@ def timestamp_to_epoch(timestamp: str) -> int:
   )
 
 
+class UnknownMachineTypeError(Exception):
+  """Defines custom exception for unknown machine-type scenario.
+
+  It holds value of machineType as str.
+  """
+
+  def __init__(self, message, machineType: str):
+    super().__init__(message)
+    self.machineType = machineType
+
+
+def resource_limits(nodeType: str) -> Tuple[dict, dict]:
+  """Returns resource limits and requests for cpu/memory for different machine types."""
+  if nodeType == "n2-standard-96":
```

Comment:
nit: Move this configuration to a yaml file, which is easier to edit for others.

Reply:
Captured in internal bug id 371077014 .